.github/workflows/deploy-pypi.yml

```diff
@@ -47,3 +44,17 @@ jobs:
       with:
         user: __token__
         password: ${{ secrets.PYPI_API_TOKEN }}
+  gh-release:
+    needs: deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set release name
+        run: echo "INPUT_NAME=v$(cat gradio/version.txt)" >> $GITHUB_ENV
+      - name: Set tag name
+        run: echo "INPUT_TAG_NAME=v$(cat gradio/version.txt)" >> $GITHUB_ENV
```

Comment:
I was thinking in case we ever wanted to name the release differently from the tag. But agree that would be a small change if we ever needed to do that so I will just use one env var!